code/CO2_monitor.ino

```diff
@@ -154,9 +169,21 @@ SCD30 airSensor;
   float humidity_web    = 0.0;
   float co2_web         = 0.0;
 
-  // create AsyncWebServer object on port 80 (port 80 for http)
-  AsyncWebServer server(80);
+  #if WIFI_WEBSERVER
+    // create AsyncWebServer object on port 80 (port 80 for http)
+    AsyncWebServer server(80);
+  #endif
+  #if WIFI_MQTT
+    WiFiClient espClient;
+    PubSubClient client(espClient);
+  #endif
+#endif
+
+#if SEND_VCC
+  ADC_MODE(ADC_VCC);
+  int vdd;
 #endif
+char buf[10];
```

Comment:
Would it be possible to move this into the `#if WIFI_MQTT` part above (line 176) ?

Also, I would like to suggest to use a more descriptive variable name, i.e. 
`char mqttMessage[10];`
or something similar.